modules/virt-about-static-and-dynamic-ssh-keys.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-accessing-vm-ssh.adoc
+
+:_content-type: REFERENCE
+[id="virt-about-static-and-dynamic-ssh-keys_{context}"]
+= About static and dynamic public keys
+
+You can add static or dynamic public SSH keys to virtual machines (VMs).
+
```

Comment:
Static and dynamic are not attributes of the public key, but attributes of the way the key is added the VM.

```suggestion
You can add public SSH keys to virtual machines (VMs) statically at the first boot or dynamically at runtime.
```